python/pyspark/tests/test_worker.py

```diff
@@ -175,13 +175,12 @@ def test_reuse_worker_of_parallelize_xrange(self):
         for pid in current_pids:
             self.assertTrue(pid in previous_pids)
 
-
 @unittest.skipIf(
     not has_resource_module,
     "Memory limit feature in Python worker is dependent on "
     "Python's 'resource' module; however, not found.")
 class WorkerMemoryTest(PySparkTestCase):
-
+    @unittest.skip("disabled temporarily since it's failing consistently")
     def test_memory_limit(self):
         self.sc._conf.set("spark.executor.pyspark.memory", "1m")
```

Comment:
The only other thing I can think of is that this toy memory limit is too low. What if it were 8m or something? if you feel like it you could try testing that too

Reply:
The odd thing is that it passes with local dev. (MacOS, python3 - 3.7.4/python2 - 2.7.10/pypy - Python 2.7.13 & pypy 7.3.0 GCC 4.2.1) so not clear it would help, but if we suspect it we can try.

Reply:
It will pass on Mac OS because Mac doesn't actually limits the memory. it doesn't work on Windows either. So, this hypothesis could make sense. Can we increase to something like 8m?

Reply:
OK let me do it instead. Let's modify the PR title/description after confirming it works.

Reply:
Maybe it's best to file a JIRA btw.

Reply:
What about filing an issue after confirming the trick doesn't work and we have to disable? We may be able to just treat this PR as MINOR but mentioning the new JIRA issue in "reason" which is expected to address this.

Reply:
Sure, that works too. FWIW, I doubt if it consistently fails in the master yet. If that's the case, it should better fix the test here.

Reply:
FYI #27159 is the patch increasing the value to 8m